step03/db.py

```diff
@@ -0,0 +1,68 @@
+from flask import g
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+import models
+from models import Profiles
+
+Base = models.Base
+
+engine = create_engine('sqlite+pysqlite:///imagely.db')
+Base.metadata.create_all(engine)
+Session = sessionmaker(bind=engine)
+session = Session()
+
+
+def registered_user(github_id):
+		try:
+				db = session
+				registered = db.query(Profiles.is_registered).filter(
```

Comment:
This can be a very expensive query on the profiles table because it will have to scan through the entire table for a match.

The better way to do this is to deal with primary keys. That way you can access the item in O(1). More on this is discussed in the comments for the Profile model